headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/dto/v1_0/converter/AttachmentDTOConverter.java

```diff
@@ -57,6 +59,11 @@ public Attachment toDTO(DTOConverterContext dtoConverterContext)
 			_cpAttachmentFileEntryService.getCPAttachmentFileEntry(
 				dtoConverterContext.getResourcePrimKey());
 
+		Company company = _companyLocalService.getCompany(
+			cpAttachmentFileEntry.getCompanyId());
+
+		String portalURL = company.getPortalURL(company.getGroupId());
```

Comment:
@andrea-ale-sbarra @marco-leo I think we may want to pass `0` for the `groupId` here. We should never include group-specific URL elements in this URL -- it should just be the company URL plus the media resolver module. `company.getGroupId` may end up doing the same thing in most cases, but it may open the door for bugs and it will almost definitely run slower than passing `0`